CONTRIBUTING.md

```diff
@@ -542,6 +543,13 @@ while lower priority or nice-to-have things may take a little longer to get appr
 To help facilitate a smoother and faster review, follow the guidelines [above](#pr-submission-guidelines).
 Submissions which do not meet standards will be de-prioritised for review.
 
+## Dependabot Bundler
+
+There is an action that will periodically bundle dependabot pull requests into a single
```

Comment:
I can extend this if more info is required. Maybe with a link to the action once it's available. Or a sample PR once there is one. :)